charts/scalardb/templates/scalardb/configmap.yaml

```diff
@@ -8,8 +8,8 @@ data:
   database.properties: |-
     scalar.db.contact_points={{.Values.scalardb.storageConfiguration.contactPoints}}
     scalar.db.contact_port={{.Values.scalardb.storageConfiguration.contactPort}}
-    scalar.db.username={{.Values.scalardb.storageConfiguration.username}}
-    scalar.db.password={{.Values.scalardb.storageConfiguration.password}}
+    scalar.db.username={{ "{{ default .Env.SCALAR_DB_USERNAME \"\" }}" }}
+    scalar.db.password={{ "{{ default .Env.SCALAR_DB_PASSWORD \"\" }}" }}
```

Comment:
These values will be evaluated as follows by Helm.
```
    scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
    scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
```

And, the above configuration (strings) will be mounted to container as `database.properties.tmpl`.
After that, the `dockerize` evaluates these values (replace to actual username and password) based on the environment variables (SCALAR_DB_USERNAME/SCALAR_DB_PASSWORD) as follows.

```
$ cat << EOF > test.properties 
scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
EOF
```
```
$ ./dockerize -template ./test.properties
scalar.db.username=
scalar.db.password=
```
```
$ export SCALAR_DB_USERNAME=test-user
$ export SCALAR_DB_PASSWORD=test-password
$ ./dockerize -template ./test.properties
scalar.db.username=test-user
scalar.db.password=test-password
```

Also, the above environment variables will be passed as Secret resource.